### DISCUSSION TITLE
DDT4ALL

### DISCUSSION THREAD

**Discussion by tatanka1969** [cedricp/ddt4all#993] [Q&A]
Good morning. Sorry for my english. I try to enter in ECRANS CONFIGURATION in DDT4ALL. I HAVE A duster EXTREME 2023. When I try to modificate I can't do anything. ANSWER ELM WRONG RESPONSE NR REQUEST OUT OF RANGE (7F2231). My OBD IS DOLLATECK WIFI

**Comment by AndyIreland** [comment]
being a 2023 car I am presuming your car has a security gateway on the OBD2 port that has to be unlocked by Renault/Dacia first before you can use/change parameters with DDT4ALL

**Reply by tatanka1969** [reply]
Everything ok, I did it🤟🏼
